src/scripts.js

```diff
@@ -5,6 +5,12 @@ import ingredientData from './data/ingredient-data';
 
 import './css/base.scss';
 import './css/styles.scss';
+import './images/apple-logo.png'
```

Comment:
Nice catch! This is via webpack import right?

Reply:
yup, i think so! In order to have access to those image you have to import them